CHANGELOG.md

```diff
@@ -1,5 +1,364 @@
 # go-ipfs changelog
 
+## v0.6.0 2020-06-16
+
+### Highlights
+
+This is a relatively small release in terms of code changes, but it contains some large changes with respect to the protocol. The highlights in this release include:
```

Comment:
Minor reword

```suggestion
This is a relatively small release in terms of code changes, but it contains some significant modifications with respect to protocols. The highlights in this release include:
```

Reply:
Does "significant changes to the IPFS protocol" make more sense?